cpp/src/arrow/engine/arrow-substrait.pc.in

```diff
@@ -23,5 +23,5 @@ Name: Apache Arrow Substrait Consumer
 Description: Apache Arrow's Substrait Consumer.
 Version: @ARROW_VERSION@
 Requires: arrow
```

Comment:
```suggestion
Requires: arrow-acero
```

Reply:
actually it requires arrow-dataset - good catch